Makefile

```diff
@@ -35,8 +35,8 @@ RUN := run
 PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
-TESTTIMEOUT  := 15s
-CPUS         := 1
+TESTTIMEOUT  := 1m
```

Comment:
Let's not make this so large; it's disconcerting. This change would remove the incentive to find out why things are slowing.